src/api/controller/api/enrichment.js

```diff
@@ -17,8 +18,8 @@ export const setup = async (ctx, next) => {
 };
 
 export const postEnrichment = async ctx => {
-    const newResource = ctx.request.body;
-    const result = await ctx.enrichment.create(newResource);
+    const newEnrichmentWithRule = await createEnrichmentRule(ctx);
```

Comment:
Shouldn't we have a try catch? `createEnrichmentRule` can raise an exception "Missing parameters"